sourcecred/sourcecred
# Add --no-zip flag to CLI, deprecate gzip suffix

# Description
This PR enables power users to disable compression of large files when using `graph` and `credrank` CLIs by adding a `--no-zip` flag. This is useful for certain non-standard but present use cases, including:
- python analysis
- using sourcecred as a backend service that pipes to other local processes


# Test Plan
1. build
2. navigate to sourcecred/cred/gh-pages clone
3. delete `output` directory and run `scdev graph --no-zip && scdev credrank --no-zip && scdev credrank -d --no-zip && scdev serve`
4. verify no errors, verify `.json` files generated in `output`, and verify cred explorer UI loads
### Regression test:
5. delete `output` directory and run `scdev graph && scdev credrank && scdev credrank -d && scdev serve`
6. verify no errors, verify compressed files generated in `output` with no file suffix, and verify cred explorer UI loads
